src/sql/parts/dashboard/common/dashboardPage.component.ts

```diff
@@ -194,10 +143,13 @@ export abstract class DashboardPage extends Disposable implements OnDestroy {
 		});
 
 		this._register(addDisposableListener(window, EventType.RESIZE, () => {
-			this._scrollableElement.setScrollDimensions({
-				scrollHeight: getContentHeight(scrollable),
-				height: getContentHeight(container)
-			});
+			// Todo: Need to set timeout because we have to make sure that the grids have already rearraged before the getContentHeight gets called.
+			setTimeout(() => {
+				this._scrollableElement.setScrollDimensions({
+					scrollHeight: getContentHeight(scrollable),
+					height: getContentHeight(container)
+				});
+			}, 100);
```

Comment:
how do we know 100ms is enough?  is there a way to make this event based to avoid needing a "magic number", or are we guaranteed that 100ms will be enough in every case?

Reply:
we don't, on super slow computers, it could be longer. Having to do this is because of how the angular2-grid behaves. We should contribute to the library so we don't need to do this.